.pylintrc

```diff
@@ -26,8 +26,9 @@ jobs=4
 # W1401(anomalous-backslash-in-string)
 # W1514(unspecified-encoding)
 # E0012(bad-option-value)
+# W1203(logging-fstring-interpolation)
```

Comment:
I think it makes sense to not disable this warning as:

1. We can save some cycles by taking advantage of the [lazy string formatting optimization](https://docs.python.org/3/howto/logging.html#optimization).
2. Keep it simple and consistent.